utils/scripts/ocm/ocm.py

```diff
@@ -267,6 +274,22 @@ def update_osd_cluster_info(self, config_file="rhoda_config_file.yaml"):
         with open(config_file, "w") as yaml_file:
             yaml_file.write(yaml.dump(config_data, default_flow_style=False))
 
+    def update_isv_cluster_info(self):
+        """Updates ISV information and stores in config file"""
+        config_file = self.configrepo_dir + "test-variables.yaml"
+        with open(config_file, "r") as fh:
+            data = yaml.safe_load(fh)
+        data["ISV_CRED"]["MONGODB"] = {}
```

Comment:
IMO, we could have the dict `data` holding `credentials` instead of prefixing `ISV`